src/deluge/model/sample/sample_recorder.h

```diff
@@ -83,19 +88,19 @@ class SampleRecorder {
 	// This will be the temp file path if there is one.
 	String filePathCreated;
 
-	uint8_t status;
+	RecorderStatus status = RecorderStatus::CAPTURING_DATA;
 	AudioInputChannel mode;
 
 	// Need to keep track of this, so we know whether to remove it. Well I guess we could just look and see if it's
 	// there... but this is nice.
-	bool haveAddedSampleToArray;
+	bool haveAddedSampleToArray = false;
 
-	bool allowFileAlterationAfter;
-	bool autoDeleteWhenDone;
+	bool allowFileAlterationAfter = false;
+	bool autoDeleteWhenDone = false;
 	bool keepingReasonsForFirstClusters;
 	uint8_t recordingNumChannels;
-	bool hadCardError;
-	bool reachedMaxFileSize;
+	bool hadCardError = false;
+	bool reachedMaxFileSize = false;
 	bool recordingExtraMargins;
 	bool pointerHeldElsewhere;
 	bool capturedTooMuch;
```

Comment:
should these be initialized too?

Reply:
probably but I don't know what they were/should be

Reply:
Yeah I think false is right, they'll all be false after a successful abort/finish recording

Reply:
I pushed a fix for this, were there any other concerns?

Reply:
not that I know of, :shipit: